README.md

```diff
@@ -129,7 +129,23 @@ compdef kubecolor=kubectl
 
 #### fish
 
-Fish completion is unsupported because fish completion is officially not supported by kubectl.
+For fish, you can achieve completions for `kubecolor` in two steps:
+
+1. Install `kubectl` completions (https://github.com/evanlucas/fish-kubectl-completions)
+
+   ```shell
+   fisher install evanlucas/fish-kubectl-completions
+   ```
+
+2. Add the following function to your `config.fish` file:
+
+   ```shell
+   function kubectl
+     kubecolor $argv
+   end
+   ```
```

Comment:
Please add indents to them because they belong to above `1. Use "fish-kubectl-completions" with kubecolor`
```suggestion
  1. Install `kubectl` completions (https://github.com/evanlucas/fish-kubectl-completions)

   ```shell
   fisher install evanlucas/fish-kubectl-completions
   ```

  2. Add the following function to your `config.fish` file:

   ```shell
   function kubectl
     kubecolor $argv
   end
   ```
```